cmd/snap/cmd_snap_op.go

```diff
@@ -948,6 +948,47 @@ func (x *cmdRevert) Execute(args []string) error {
 	return nil
 }
 
+var shortSwitchHelp = i18n.G("Switch snap to a different channel")
```

Comment:
s/Switch/Switches/

We're still somewhat inconsistent about the phrasing of help text, but it'd be nice to fix it eventually. To that end, I've brought back this ancient thread from the old mailing list:

https://forum.snapcraft.io/t/convention-for-command-help-text/1856